_docs/concepts/auth.md

```diff
@@ -0,0 +1,138 @@
+---
+category: Concepts
+title: Auth
+overview: Architectural deep-dive into the design of Auth, which provides the secure communication channel and strong identity for Istio.
+              
```

Comment:
You don't need category: and parent: anymore. order: is still used and determines the ordering of items in the left-hand nav and in generated index pages.